util/throttle.cc

```diff
@@ -168,6 +179,7 @@ ThrottleThread(
 	else if (0!=gThrottleData[0].m_Keys && 0!=gThrottleData[0].m_Compactions)
 	{
             pthread_mutex_lock(&gThrottleMutex);
+
             new_throttle=(gThrottleData[0].m_Micros / gThrottleData[0].m_Keys)
```

Comment:
We have a potential divide-by-0 here if someone calls SetThrottleWriteRate() after we check gThrottleData[0].m_Keys and gThrottleData[0].m_Compactions but before we lock gThrottleMutex.